crates/winrt-build/Cargo.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "winrt-build"
+version = "0.1.0"
+authors = ["Ryan Levick <ryan.levick@gmail.com>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
```

Comment:
For consistency with the other toml files, should we set the author to "Microsoft" and remove this comment?